src/private/MDMPerformerGroup.m

```diff
@@ -72,6 +72,19 @@ - (void)executeLog:(MDMTransactionLog *)log trace:(MDMTrace *)trace {
     if ([performer respondsToSelector:@selector(addPlan:)]) {
       [(id<MDMPlanPerforming>)performer addPlan:plan];
     }
+    
+    // MDMNamedPlanPerforming callbacks
+    if (log.name.length) {
```

Comment:
Assuming we support empty names, presence of a non-nil `name` property will be a sufficient check here.